Cargo.toml

```diff
@@ -14,13 +14,13 @@ repository = "https://github.com/dylanhart/ulid-rs"
 
 [features]
 default = ["std"]
-std = ["rand", "time"]
+std = ["rand"]
+time = []
```

Comment:
Why was `time` added as a feature here? I'm not seeing anything that uses it. I think we can remove this and the reference to `time` (the library) from README.md. :)

Reply:
I'd thought this is required as users may have selected: `features = ["time"]`, however on second thoughts, this is a semver incompatible change anyway so it's fine to have breakage here as well